### DISCUSSION TITLE
Jenkins Multibranch configuration with Bitbucket Branch source plugin supported?

### DISCUSSION THREAD

**Discussion by chrissike** [github/gh-actions-importer#78] [Feedback]
Dear owners,

first of all: I'm super excited about the actions importer! It has the potential to be an absolute game changer on our current journey. We're migrating 1000+ projects from Bitbucket+Jenkins to GitHub. So currently the Jenkins pipelines we need to migrate are all configured to get their code and Jenkinsfiles from Bitbucket.

After some first tests I'm running into issues with our Jenkins Multibranch pipelines. 
The connection to Bitbucket is configured with the Bitbucket branch source plugin. I'm getting errors, that the credential for the Bitbucket repo can't be retrieved, and then I guess the rest of it fails because the Jenkinsfile isn't available for analysis. 

I assume the actions importer currently is built for teams who have already been using GitHub as their code repository and now just want to move their CI/CD part to GitHub as well. However I think for many GitHub Enterprise customers it would be a big win to also be able to import Jenkins pipelines with other source types. 

Maybe I'm wrong with my assumption and I'm just doing something wrong, that would make me very happy :) so any hints welcome!

BR Christina

**Comment by assyadh** [comment]


**Reply by chrissike** [reply]
Hi @assyadh,
thank you for your quick reply!

We're actually using Bitbucket Server. Within the multibranch pipelines we use this plugin, to configure our Bitbucket server repos as a source: https://plugins.jenkins.io/cloudbees-bitbucket-branch-source/ 

I'll look into the docs you've linked, thanks for sharing!

**Reply by stves** [reply]
Hey @chrissike, I'm going to mark @assyadh's response as the answer to this discussion. But please let us know if that didn't solve your problem!
